### ISSUE TITLE
Survival Data Crashes Website

### ISSUE THREAD

**Issue by Huh** [Huh/PopR_SDGFP#96]
At the moment loading survival data will crash the website.  I am investigating the issue and will have a solution tomorrow afternoon at the latest whether it be data or code based.

Sorry for the inconvenience. 

Josh

**Comment by Huh** [comment]
Cannot find any age entries for the following animals.

```r
           ID capage_fun(Ageclass)
        (chr)                (dbl)
1  2016422002                   NA
2  2016422005                   NA
3  2016422011                   NA
4  2016422013                   NA
5  2016422014                   NA
6  2016422015                   NA
7  2016422018                   NA
8  2016422019                   NA
9  2016422020                   NA
10 2016422023                   NA
11 2016422024                   NA
12 2016422027                   NA
13 2016422028                   NA
```

I will work on making the code more resilient to this type of error, but in the meantime it would be good to fix the data and give each of these animals an age.

**Comment by gfrc11752** [comment]
This issue should be resolved. Good catch…

**Comment by Huh** [comment]
Need to improve the way errors are propagated in this tab.  The error above should not have crashed the website.  There was one place where a reactive element was null and uncaught.

**Comment by gfrc11752** [comment]
Quick question- age class is only assigned from the capture encounter history correct? I am thinking about getting rid of age class in the monitoring data collection process because PopR should already be assigning the age category depending on the capture record.

**Comment by Huh** [comment]
Correct.  I don't see adding age to monitoring data as a useful redundancy.  I only see it creating problems.  The code goes through each age class from youngest to oldest and only retains the youngest ageclass assigned to the animal.   Josh Nowak 

    On Wednesday, March 23, 2016 11:45 AM, gfrc11752 <notifications@github.com> wrote:
 

 Quick question- age class is only assigned from the capture encounter history correct? I am thinking about getting rid of age class in the monitoring data collection process because PopR should already be assigning the age category depending on the capture record.

**Comment by gfrc11752** [comment]
Very good, thanks!
